qemu/tests/blockdev_mirror_complete_running_job.py

```diff
@@ -0,0 +1,49 @@
+from virttest.qemu_monitor import QMPCmdError
+
+from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
+
+
+class BlockdevMirrorCompleteRunningJobTest(BlockdevMirrorNowaitTest):
+    """
+    Complete a running blockdev-mirror job
+    """
+
+    def cancel_running_mirror_job(self):
```

Comment:
Suggest to change the function name to 'complete_running_mirror_job'.

Reply:
done